### PR TITLE
s3cmd 1.5

### PR DESCRIPTION
**Pull request by billywatson**
Bump s3cmd to 1.5 and remove 1.5.0-rc1 devel version

### CODE REVIEW

**Comment by billywatson** [comment]
Just updated and fixed. I just figured out how I can test it myself using brew edit so the first version was awful.

File: Library/Formula/s3cmd.rb [hunk 1]
```diff
@@ -2,24 +2,10 @@
...
-    version "1.5.0-rc1"
-  end
-
-  bottle do
-    cellar :any
-    revision 1
-    sha1 "7493692a3dc0b34a8d3a617bd4e36df9fed16226" => :mavericks
-    sha1 "5161ab336e7f34d8442b4a0c5964a99004ff03d9" => :mountain_lion
-    sha1 "198a88c5490cdc705fee324a131a44360c826f63" => :lion
-  end
```
Comment (mikemcquaid) [inline Library/Formula/s3cmd.rb]: Just leave the bottle block as-is and we'll update it when we pull.

File: Library/Formula/s3cmd.rb [hunk 2]
```diff
@@ -2,24 +2,10 @@
 
 class S3cmd < Formula
   homepage 'http://s3tools.org/s3cmd'
-  url 'https://github.com/s3tools/s3cmd/archive/v1.0.1.tar.gz'
-  sha1 '98ef9ea9e2dc56ab98f84610883d3d371c65e43a'
+  url 'http://downloads.sourceforge.net/project/s3tools/s3cmd/1.5.0/s3cmd-1.5.0.tar.gz'
```
Comment (mikemcquaid) [inline Library/Formula/s3cmd.rb]: Use https: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/19951/version=yosemite/testReport/junit/brew-test-bot/yosemite/audit_s3cmd/

**Comment by billywatson** [comment]
Updated again according to the review. Sorry it took me so long to get back to it

**Pull request closed without merging**